src/renderer/features/wallets/ShardSelectorModal/ui/SelectableShard.tsx

```diff
@@ -35,15 +35,11 @@ export const SelectableShard = ({
   const isSharded = isShard || isChain;
   const address = toAddress(account.accountId, { prefix: addressPrefix });
 
+  const theme = isBase && walletUtils.isPolkadotVault(wallet) ? 'jdenticon' : undefined;
```

Comment:
Let's pass `polkadot` instead of `undefined`, otherwise it looks awkward

Reply:
In that case we need not only `polkadot` but `ethereum` too. `Identicon` component already can select default theme. I can't see any reasons to check it twice.